### DISCUSSION TITLE
SQL資料庫基礎教學資源分享

### DISCUSSION THREAD

**Discussion by manipulateed** [1101-GS4719-C/Python-Community-GS4719-C#730] [General]
透過這個影片讓我在程式設計的專題中，對SQL的基礎語法運用自如，算是講得非常易懂，分享給大家
https://www.youtube.com/watch?v=gvRXjsrpCHw

**Comment by williamyen0623** [comment]
SQL的應用非常廣泛，值得學習，感謝分享！

**Comment by Zoe06060606** [comment]
之前也修過網路相關的程式可能有稍微碰過SQL 但仍不太熟悉 這個影片就闡述了很多相關用法 但又不會太難懂👍

**Comment by danwan1027** [comment]
資料庫處理對資料分析也有幫助 感謝分享！

**Comment by evanchou1204** [comment]
我讀財管系的朋友說他自學SQL，他覺得SQL比for迴圈簡單???

**Comment by Yunhans** [comment]
之前上ai人工智慧的時候看SQL是一知半解，不知道之後的課程會不會學到
